### DISCUSSION TITLE
Раздел "Что нового"?

### DISCUSSION THREAD

**Discussion by vitalybaev** [doka-guide/platform#867] [Идеи]
По мотивам:

https://twitter.com/LollipopCha1n/status/1481847452596682752

Добавлю от себя:

Несмотря, что Дока - чаще всего про историю, когда тебе нужно прочитать конкретную тему, я лично для себя вижу пользу в том, чтобы читать новые написанные статьи. Особенно это актуально в текущей ситуации, когда в Доке есть еще много непокрытых тем.

Сейчас единственный способ этого добиться - следить за соцсетями Доки или же открывать `CHANGELOG.md` в репозитории.

Как я это вижу? Да в принципе примерно так же, как список [статей на сайте Веб-стандартов](https://web-standards.ru/articles/). Единственное, в Доке точно нужна пагинация или, возможно, разбиение по годам или около того. Или вообще N последних материалов.

**Comment by Inventoris** [comment]
@vitalybaev, привет! Похоже, Дока скоро получит эту важную фичу. У команды в работе PR #1286 (присоединяйся к ревью). Как только его закончат, на главной появится ченджлог как раз в формате N последних материалов!
